index.html

```diff
@@ -500,19 +502,31 @@ <h4>Script Type</h4>
                 <p>It can be adapted to produce a <a>Pre-Recording Script</a>,
                   and/or used as the basis for a further translation into the <a>Target Recording Language</a>.</p> 
                 <p><a>Script Events</a> in this type of <a>transcript</a> SHOULD contain <a>Text</a> objects whose
-                  <a>Text Language Source</a> is set to <a>Translation</a>.
-                  They MAY also contain <a>Original</a> language <a>Text</a>.</p>
+                  <a>Text Language Source</a> is set to the source language from which they were translated,
+                  and whose language is set to their actual language,
+                  indicating that they are <a>Translations</a>.
+                  They MAY also contain <a>Original</a> <a>Text</a> objects.</p>
```

Comment:
Should we say that:
* All events SHALL have at least one text object with the same language. 
* That language MAY or MAY NOT be an original language. 
* Other text objects MAY exist marked with an (other) original language.

For example if the original languages are English and French, if the translation language is French, all events will include French text objects and may include English text objects. But if the translated language is German, all events will include German text objects and may include English and French text objects.

Reply:
I don't follow: we don't have explicit metadata indicating what is the Target Recording Language, so I don't think that those rules would be testable. Isn't the text in this PR saying what you mean though? Possibly there's a requirement I haven't captured or understood so far.